app/models/financial_aid/my_awards.rb

```diff
@@ -178,29 +178,33 @@ def link_params
     end
 
     def outside_resources_available?
-      exists ||= EdoOracle::FinancialAid::Queries.get_awards_outside_resources(my_aid_year) 
-      exists ? true : false
+      @outside_resources_available ||= {}
```

Comment:
@marcpechaitis - It doesn't make sense to initialize this as a Hash like this. This can only result in the variable being returned as an empty hash, without the query being called at all. I'd simply remove line 181, because 183 is sufficient enough. You only need the hash if the method is dynamic based on some other variable input like a method argument.